python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -524,6 +531,72 @@ def resolve_hasResumePermission(self, graphene_info: ResolveInfo) -> bool:
     def resolve_user(self, _graphene_info: ResolveInfo) -> Optional[str]:
         return self._backfill_job.user
 
+    def resolve_logEvents(self, graphene_info: ResolveInfo, cursor: Optional[str] = None):
+        # very unsure how the cursor gets passed through, especially if called from get_backfills which also has a cursor
+        from ..schema.instigation import (
+            GrapheneInstigationEvent,
+            GrapheneInstigationEventConnection,
+        )
+        from ..schema.logs.log_level import GrapheneLogLevel
+
+        backfill_id = self._backfill_job.backfill_id
+        # TODO find a better way to keep this in sync. maybe store as part of the asset backfill data?
+        backfill_log_key_prefix = ["backfill", backfill_id]
+
+        instance = graphene_info.context.instance
+
+        if not isinstance(instance.compute_log_manager, CapturedLogManager):
+            return GrapheneInstigationEventConnection(events=[], cursor="", hasMore=False)
+
+        # TODO - need to gate to plus only
+
+        log_keys = sorted(
+            instance.compute_log_manager.get_log_keys_for_log_key_prefix(
```

Comment:
`get_log_keys_for_log_key_prefix` is implemented in internal pr. it basically looks at all of the files in the directory and creates the log key for each file

Reply:
might make sense to push a lot of this logic down in to the log manager and just handle translation at the graphql layer